lib/beaker/host/cisco.rb

```diff
@@ -72,7 +72,7 @@ def prepend_commands(command = '', user_pc = '', opts = {})
       return user_pc unless command.index('vsh').nil?
 
       prepend_cmds = 'source /etc/profile;'
-      if self[:platform] =~ /cisco-5/
+      if self[:platform] =~ /cisco-nexus/
```

Comment:
I think you have an older 'beta' IOS XR VM at present. There've been a number of enhancements and recommendations for the workflow that have been made in the last few months. I'll make sure we cover this in our next interlock meeting with PuppetLabs. Thanks for the details - these will be helpful.